buildpipeline/DotNet-CoreFx-Trusted-Linux-Crossbuild.json

```diff
@@ -179,30 +199,57 @@
       }
     },
     {
+      "environment": {},
       "enabled": true,
       "continueOnError": true,
-      "alwaysRun": true,
-      "displayName": "Copy Publish Artifact: BuildLogs",
+      "alwaysRun": false,
+      "displayName": "Copy Files to: $(BUILD_STAGINGDIRECTORY)\\BuildLogs",
       "timeoutInMinutes": 0,
+      "refName": "CopyFiles1",
       "task": {
-        "id": "1d341bb0-2106-458c-8422-d00bcea6512a",
-        "versionSpec": "1.*",
+        "id": "5bfb729a-a7c8-4a78-a7c3-8d717bb7c13c",
+        "versionSpec": "2.*",
         "definitionType": "task"
       },
       "inputs": {
-        "CopyRoot": "$(PB_GitDirectory)",
+        "SourceFolder": "$(PB_GitDirectory)",
         "Contents": "*.log",
+        "TargetFolder": "$(BUILD_STAGINGDIRECTORY)\\BuildLogs",
+        "CleanTargetFolder": "false",
+        "OverWrite": "false",
+        "flattenFolders": "false"
+      }
+    },
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": false,
+      "displayName": "Publish Artifact: BuildLogs",
+      "timeoutInMinutes": 0,
+      "refName": "PublishBuildArtifacts2",
+      "task": {
+        "id": "2ff763a7-ce83-4e1f-bc89-0ae63477cebe",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "PathtoPublish": "$(BUILD_STAGINGDIRECTORY)\\BuildLogs",
         "ArtifactName": "BuildLogs",
         "ArtifactType": "Container",
-        "TargetPath": "\\\\my\\share\\$(Build.DefinitionName)\\$(Build.BuildNumber)"
+        "TargetPath": "\\\\my\\share\\$(Build.DefinitionName)\\$(Build.BuildNumber)",
+        "Parallel": "false",
+        "ParallelCount": "8"
       }
     },
     {
+      "environment": {},
       "enabled": true,
       "continueOnError": true,
-      "alwaysRun": true,
+      "alwaysRun": false,
```

Comment:
Why this change? Don't we need to cleanup docker any more?

Reply:
I'm not certain where this change came from.